pytravharv/common.py

```diff
@@ -0,0 +1,66 @@
+import os
+from logging import getLogger
+from urllib.parse import quote, unquote
+
+import validators
+from pyrdfj2 import J2RDFSyntaxBuilder
+from rdflib import Graph
+
+from pytravharv.WebAccess import fetch
+
+log = getLogger(__name__)
+
+QUERY_BUILDER = J2RDFSyntaxBuilder(
+    templates_folder=os.path.join(
+        os.path.dirname(os.path.realpath(__file__)), "templates"
+    )
+)
+
+
+def graph_name_to_uri(graph_name: str) -> str:
```

Comment:
it is entirely unclear what is the semantics and expected use of these two convertor methods

maybe take some inspiration from: https://github.com/vliz-be-opsci/py-sync-fs-triples/blob/main/syncfstriples/service.py#L25

that also allows for having a `base` to be injected

make sure you have clear semantics separating the two worlds you are converting between

- on the one hand you need a `ng` == named graph (uri) to be used in the rdf_store api
- on the other hand I think you have some trav-harv name/identifier thingy that comes from the config? task-context?

so maybe name things `GraphTaskNameMapper` with `ng_to_taskname(ng)` versus `taskname_to_ng(taskname)`

Reply:
issue mentioned in #18